addons/Yush/Yush.lua

```diff
@@ -75,12 +77,24 @@ output = function()
     if settings.Verbose then
         names[current] = names[current] or 'Unnamed ' .. tostring(current):sub(8)
 
+        output_text_table = {}
```

Comment:
This variable should be `local`. I also prefer `output_text_names` over `_table`, which is a bit too generic, and then you could make it a list (`= L{}`) without mangling the meaning of the name. And a list is more fitting here, will make the next part of the code a bit easier.